ex1/ex1Spec.js

```diff
@@ -0,0 +1,19 @@
+var getThirdCommaIndex = require("./ex1");
+
+describe("getThirdCommaIndex", function() {
+
+	it("returns -1 when no third comma", function() {
+		expect(getThirdCommaIndex('hel,lowo,rld', ','))
+		.toBe(-1);
+	});
+
+	it("finds a comma in a regular sentence", function() {
+		expect(getThirdCommaIndex('hello,world,this,is,a,great,day', ','))
```

Comment:
The maximum line length on github.com is 80 character, so this line gets too long. You could break this into two lines, but I think it might be easier to use a shorter tab (maybe two spaces instead of an entire tab? See https://www.google.com/search?sourceid=chrome-psyapi2&ion=1&espv=2&ie=UTF-8&q=vim%20tabs%20to%20spaces&oq=vim%20tab&aqs=chrome.1.69i57j0l5.3624j0j7